libs/jsonschema/schema.go

```diff
@@ -87,6 +88,43 @@ type Schema struct {
 	Deprecated bool `json:"deprecated,omitempty"`
 }
 
+func anyToSchema(a any) (*Schema, error) {
```

Comment:
Why is this needed? As I understand it, the leafs of the definitions structure are always a schema.

Reply:
Yes, but when loading the schema itself via a `JSON.Unmarshall` call, the leaf nodes are not converted to a schema type in Go. They still are a `map[string]any` in the in-memory data structure.

This function is needed to convert the leaf node from a map to a schema object.

Reply:
Please include this as a comment. Good context for the reader.